console/v1/types_console_link.go

```diff
@@ -0,0 +1,41 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Extension for customizing OpenShift web console links
+type ConsoleLink struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ConsoleLinkSpec `json:"spec"`
+}
+
+type ConsoleLinkSpec struct {
+	Link
```

Comment:
missing `json:`?

Reply:
`Link` is `embedded` https://github.com/openshift/api/pull/318/files#diff-ab660a922bf15f67f51080851b29db7aR4.

Reply:
ah, yeah `json:",inline"`, yup thx!